src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -63,12 +77,7 @@ using namespace chip::app::Clusters;
       ^(Cancelable * success, Cancelable * failure) {
         auto successFn = Callback<CHIP{{>callbackName}}CallbackType>::FromCancelable(success);
         auto failureFn = Callback<CHIPDefaultFailureCallbackType>::FromCancelable(failure);
-        return self.cppCluster.InvokeCommand(request, successFn->mContext, successFn->mCall, failureFn->mCall
-        {{#if mustUseTimedInvoke}}
```

Comment:
Should we perhaps continue providing a default somewhere here?

That is, it seems like by default we should send valid commands.  I guess we want to support sending invalid ones for chip-tool-darwin to run the test suite, but we should make it require some effort to send the invalid ones...

If we made the timed invoke bits part of the params, we could have the params init do this defaulting, for the commands that `mustUseTimedInvoke`, right?

Or is the idea that non-chiptool consumers would not even use the WithArguments API directly?  But then what happens when we add the other things we wanted to add there?

Reply:
I added this to the legacy function. I didn't make the classes have a default value, because every command will trigger a timed command/write request rather than actually doing the command.

Reply:
I guess I am trying to understand which API consumers are expected to use which API here and how....

Reply:
I am thinking that we should deprecate the old way of doing it and have all clients use the additional parameter or set it to nil if the do not want to pass in any parameters. What do you think?

Reply:
We spoke offline and the best course of action is not to allow invalid timed command or invalid time write. If not timed write is passed in, then we should always default to 10000. Updated Pr.